src/urbit/app/chess.hoon

```diff
@@ -90,6 +90,9 @@
               "already challenged {<who.action>}"
             :~  [%give %poke-ack `~[leaf+err]]
             ==
+          ::  XX: add error on attempting to send a
+          ::      challenge to yourself
+          ::
```

Comment:
That's actually not necessary - I often check changes in my fake ships by sending a challenge to myself and playing both sides in one interface